runtime/repl.go

```diff
@@ -66,15 +67,18 @@ func NewREPL(
 
 	storage := interpreter.NewInMemoryStorage(nil)
 
-	// NOTE: storage option must be provided *before* the predeclared values option,
-	// as predeclared values may rely on storage
+	// necessary now due to log being looked up in the
+	// interpreter's activations instead of the checker
+	baseActivation := interpreter.NewBaseActivation()
+	interpreter.Declare(baseActivation, stdlib.NewLogFunction(cmd.StandardOutputLogger{}))
```

Comment:
Maybe add a comment to the equivalent declaration of for the checker in `DefaultCheckerConfig`:
https://github.com/onflow/cadence/blob/e2b6a824df0cad26f60d493e9113acef70c2a909/runtime/cmd/cmd.go#L88

```
// NOTE: also ensure to implement all definitions, e.g. for the `REPL` in `NewREPL`
```